examples/shardingsphere-sample/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/scenario/ExampleScenarioFactory.java

```diff
@@ -86,10 +91,29 @@ private FrameworkExampleScenario getFrameworkScenario(final String framework) {
     public Map<String, String> getResourceTemplateMap() {
         Map<String, String> result = new HashMap<>();
         for (FeatureExampleScenario each : featureScenarios) {
-            result.putAll(each.getJavaClassTemplateMap());
+            result.putAll(each.getResourceTemplateMap());
         }
         result.putAll(frameworkScenario.getResourceTemplateMap());
         result.put("resources/logback.ftl", "logback.xml");
         return result;
     }
+    
+    public Set<String> getJavaClassPathSet() {
```

Comment:
1. `getJavaClassPathSet` is not a good name, `Set` is for implementation, the code user do not need care about it, please just name it as `getJavaClassPaths`.
2. Use Collection to instead of Set if passable.